libsolutil/Whiskers.cpp

```diff
@@ -48,6 +54,7 @@ Whiskers& Whiskers::operator()(string _parameter, bool _value)
 {
 	checkParameterValid(_parameter);
 	checkParameterUnknown(_parameter);
+	checkTemplateContainsTags(_parameter, {"?", "/"});
```

Comment:
Is this necessarily the case? I mean must there be tags `<?a>` and `</a>` if I add `a`? Just wondering if this couldn't for example just be any string placement

Reply:
Yes, it's a bool parameter, I think it can only be accessed using `<?a>...</a>`.

Reply:
Ok